arcade/arcade/templates/{{ toolkit_name }}/.ruff.toml

```diff
@@ -0,0 +1,62 @@
+target-version = "py39"
+line-length = 100
+fix = true
+
+[lint]
+select = [
+    # flake8-2020
+    "YTT",
+    # flake8-bandit
+    "S",
+    # flake8-bugbear
+    "B",
+    # flake8-builtins
+    "A",
+    # flake8-comprehensions
+    "C4",
+    # flake8-debugger
+    "T10",
+    # flake8-simplify
+    "SIM",
+    # isort
+    "I",
+    # mccabe
+    "C90",
+    # pycodestyle
+    "E", "W",
+    # pyflakes
+    "F",
+    # pygrep-hooks
+    "PGH",
+    # pyupgrade
+    "UP",
+    # ruff
+    "RUF",
+    # tryceratops
+    "TRY",
+]
+
+# TODO work to remove these
+ignore = [
+    # LineTooLong
+    "E501",
+    # DoNotAssignLambda
+    "E731",
+    # raise from (cli specific)
+    "B904", # Previously "TRY200"
+    # Depends function in arg string
+    "B008",
+    # raise from (cli specific)
+    "B904",
+    # long message exceptions
+    "TRY003",
+    # subprocess.Popen
+    "S603",
+]
```

Comment:
Could we remove these ignores? (I have been meaning to remove it in our own project too)

However, double check that removing this doesn't cause a new toolkit to immediately fail the stricter linting rules.

If some of these ignores need to remain, then remove the `# TODO work to remove these` part only 😛

Reply:
Removed with no issues with `make check` 😀

Reply:
Woot